libs/base/Language/Reflection/TT.idr

```diff
@@ -108,7 +108,7 @@ data NameType : Type where
      Bound   : NameType
      Func    : NameType
      DataCon : (tag : Int) -> (arity : Nat) -> NameType
-     TyCon   : (tag : Int) -> (arity : Nat) -> NameType
```

Comment:
Given how work intensive it is to make such changes to elab-reflection, what do you think of reverting that and keeping the dummy tag around until we collect more changes and update elab reflection later?